lib/radiator_web/views/api/chapters_view.ex

```diff
@@ -1,14 +1,22 @@
 defmodule RadiatorWeb.Api.ChaptersView do
   use RadiatorWeb, :view
 
-  alias HAL.{Document, Link}
+  alias HAL.{Document, Link, Embed}
   alias Radiator.AudioMeta.Chapter
 
+  def render("index.json", assigns) do
+    %Document{}
+    |> Document.add_embed(%Embed{
+      resource: "rad:chapter",
+      embed: render_many(assigns[:chapters], __MODULE__, "chapter.json", assigns)
+    })
+  end
+
   def render("show.json", assigns) do
     render(__MODULE__, "chapter.json", assigns)
   end
 
-  def render("chapter.json", %{conn: conn, chapter: chapter, audio: audio}) do
+  def render("chapter.json", %{conn: conn, chapters: chapter, audio: audio}) do
```

Comment:
why the key chapters if you only render one?, shouldn't it still be chapter?

Reply:
Yep, fixed already. The `render_many` put the chapter under the `chapters` key for some reason, so I rewrote it with a manual `Enum.map` and now I can use the `chapter` key again.